### ISSUE TITLE
did 2.0.2 just kill paths

### ISSUE THREAD

**Issue by mattstyles** [plus3network/gulp-less#138]
```
.pipe( $.less({
            paths: [
                '.',
                './src/vendor/'
            ]
        }))
```

`lesshat` currently lives in `./src/vendor/` but 2.0.2 gives me the error:

```
[09:41:58] Plumber found unhandled error:
 Error in plugin 'gulp-less'
Message:
    'lesshat/build/lesshat.less' wasn't found. Tried - /Users/mattstyles/projects/myprojects/koa/front-end/src/styles/lesshat/build/lesshat.less,lesshat/build/lesshat.less in file /Users/mattstyles/projects/myprojects/koa/front-end/src/styles/main.less line no. 2
Details:
    type: File
    filename: /Users/mattstyles/projects/myprojects/koa/front-end/src/styles/main.less
    index: 1
    line: 2
    callLine: NaN
    callExtract: undefined
    column: 0
    extract: ,@import 'lesshat/build/lesshat';,
    lineNumber: 2
    fileName: /Users/mattstyles/projects/myprojects/koa/front-end/src/styles/main.less
```

just reverted back to 2.0.1 and all is happy again.

is this expected and I've just missed how to do this?

**Comment by harisenbon** [comment]
I'm having the same issue. Multiple paths are completely dead. Had to revert to 2.0.1.

Message:
'home/landing.less' wasn't found. Tried - /srv/vhosts/******/releases/20150211100727/vendor/bower_components/bootstrap/less/home/landing.less,home/landing.less in file /srv/vhosts/******/releases/20150211100727/vendor/bower_components/bootstrap/less/home.css line no. 66

**Comment by OhJeez** [comment]
If people who do maintain this package would read contributions, that would not have happened, as I was warning about it in #131 and #132 .
Also, gulp-less still does not use promises properly.

Should I, or shouldn't I fork it, that is the question.

**Comment by DronRathore** [comment]
Fixed this in #139 , waiting for merge.

**Comment by contra** [comment]
@OhJeez Send a PR or I'll just give you this module. I got stuck maintaining it because the original author got busy, and now I'm busy and don't have a ton of time to really look through every single PR or issue

**Comment by contra** [comment]
Fixed in 2.0.3
